plot/plot.py

```diff
@@ -63,38 +67,83 @@ def parse_data(csv_path):
 
 
 def save_plot(df, out_path):
-    DAYS = df["weekday"].unique()
-
     # Pick an appropriate y-axis, balancing being robust to outliers vs. showing all data
     Y_MAX = df["solve_time_secs"].quantile(0.99) / 60
+    Y_MAX = 60 # unfortunately, an hour is a good upper limit for me.
```

Comment:
Can you make this an optional argument that can override the default auto-ranging behavior?

Reply:
done, good idea!